packages/gatsby-plugin-theme-ui/README.md

```diff
@@ -3,7 +3,7 @@
 Gatsby plugin for adding theme-ui context
 
 ```sh
-npm i theme-ui gatsby-plugin-theme-ui @emotion/core @mdx-js/react
+npm i theme-ui gatsby-plugin-theme-ui @emotion/core @mdx-js/react @theme-ui/presets
```

Comment:
Thanks for this! I think it'd be okay to leave the presets package out of the main install instructions (since it's not required for the plugin), and maybe mention it below where it's used
```suggestion
npm i theme-ui gatsby-plugin-theme-ui @emotion/core @mdx-js/react
```